doc/classes/GPUParticles2D.xml

```diff
@@ -71,7 +71,8 @@
 			Particle texture. If [code]null[/code], particles will be squares.
 		</member>
 		<member name="visibility_rect" type="Rect2" setter="set_visibility_rect" getter="get_visibility_rect" default="Rect2( -100, -100, 200, 200 )">
-			Editor visibility helper.
+			The [Rect2] that determines the area of the world part of which needs to be visible on screen for the particle system to be active.
```

Comment:
"the area of the world part of which" sounds weird.

Should it just be "the area of the world which"? (same for the original string copied from the 3D one)

Reply:
Yes, I thought the same while copying the 3D one.

Could also be:

```suggestion
			The [Rect2] that determines the node's area which needs to be visible on screen for the particle system to be active.
```

Reply:
Sounds good yeah, "world" can also be confused with World2D/World3D.